### ISSUE TITLE
Is it possible to scroll inside the schedule instead of only on days and hours labels?

### ISSUE THREAD

**Issue by ryanhz** [pebble-world/planner#1]
Great plugin. Is it possible to scroll inside the schedule instead of only on days and hours labels?  Intuitively, we just want to move around inside the planner.

**Comment by yvanvds** [comment]
You are right, it would be more intuitively. But it would mean that panning and scrolling would act on the same canvas. I've tried implementing that, but it didn't work too well. When I wanted to pan, more often than not the scrolling was also triggered, and it felt very clumsy overall.

Still, I will probably give it another go over the summer holidays and see if I can come up with another way to tackle this problem.
